src/com/actelion/research/gwt/gui/generic/JSClipboardHandler.java

```diff
@@ -19,72 +21,70 @@ private JavaScriptObject getJsHandler() {
     return mJsHandler;
   }
 
-  public native StereoMolecule pasteMolecule()
+  public native boolean putStringJS(String data)
   /*-{
-    var handler = this.@com.actelion.research.gwt.gui.generic.JSClipboardHandler::getJsHandler()();
-    var jsMolecule = handler.pasteMolecule();
-    if (jsMolecule) {
-      return jsMolecule.@com.actelion.research.gwt.minimal.JSMolecule::getStereoMolecule()();
-    } else {
-      return null;
-    }
+    var clipboard = this.@com.actelion.research.gwt.gui.generic.JSClipboardHandler::getJsHandler()();
+    return clipboard.putString(data);
   }-*/;
 
+  // Can't be implemented here
+  // Clipboard is not accessible with sync API
+  public StereoMolecule pasteMolecule() {
+    return null;
+  }
+
+  // Not used, No implementation needed yet
   public StereoMolecule pasteMolecule(boolean prefer2D, int smartsMode) {
-    Object test = null;
-    System.out.println("test: " + test.toString());
```

Comment:
`test.toString()` produce an error (test is null).

Theses errors was really annoying to implement copy-paste properly.

So I just changes theses method so it's a no-op